store/engine/consul/consul.go

```diff
@@ -209,7 +214,7 @@ func (c *Consul) List(ctx context.Context, prefix string) ([]engine.Entry, error
 		if string(kv.Key) == prefix {
 			continue
 		}
-		key := strings.TrimLeft(string(kv.Key[prefixLen+1]), "/")
+		key := strings.TrimLeft(string(kv.Key[prefixLen+1:]), "/")
```

Comment:
Bug #2 fix. I noticed that this was missing the `:` which I could find in the `etcd` and other implementations